docs/src/code/style-guide.adoc

```diff
@@ -1,8 +1,27 @@
 = Coding Style
 
-This chapter describes the source code style preferred by the LinuxCNC team.
+LinuxCNC is developed in C/C++, Python, BASH and Tcl/TK.
+Documentation is written in asciidoc, which includes parts of the
+UNIX man pages.
```

Comment:
I don't understand the "includes parts of of the UNIX man pages" !? Do you mean that parts of the documentation are in form of man pages ?

Reply:
Yes. For most of the small Hal components there is only documentation available as man page.

Reply:
> po4a could not handle the alt image descriptions

I have circumvented much of the "alt=" instances in the docs, but thinking about it and having read a bit more about it in the asciidoc documentation, this likely is more of a po4a bug.